daisy_workflows/build-publish/enterprise_linux/almalinux_8.wf.json

```diff
@@ -31,6 +31,10 @@
     "syft_source": {
       "Value": "",
       "Description": "Source url for the syft tar gz file, if generating SBOM. If empty no SBOM is generated."
+    },
+    "sbom_util_source": {
```

Comment:
We are using `sbom_util_gcs_root` for packages, it'd be nice if we kept them consistent.

Reply:
done